ios/Podfile

```diff
@@ -21,7 +21,7 @@ target 'BitriseReactNativeSample' do
     #
```

Comment:
Noticed this warning in the build logs, you might want to address this:

```
[!] Your project does not explicitly specify the CocoaPods master specs repo. Since CDN is now used as the default, you may safely remove it from your repos directory via `pod repo remove master`. To suppress this warning please add `warn_for_unused_master_specs_repo => false` to your Podfile.
```

Reply:
Hmm this may be an issue on the stack. I don't have that repo locally. I guess I can suppress the warning.

Reply:
44bfcd0